provisioning_templates/snippet/preseed_networking_setup.erb

```diff
@@ -8,9 +8,9 @@ description: this will configure your host networking, it configures your primar
     called in your preseed finish template.
 %>
 <% host_subnet = @host.subnet -%>
-<% host_dhcp = host_subnet.nil? ? false : host_subnet.dhcp_boot_mode? -%>
+<% host_dhcp = host_subnet.nil? ? true : host_subnet.dhcp_boot_mode? -%>
```

Comment:
@lzap I've also just noticed that preseed_default.erb [passes the `static` parameter onto the finish template](https://github.com/theforeman/community-templates/blob/44a23f5d51982e8b3c0e30c87b5a5a02c2a33950/provisioning_templates/provision/preseed_default.erb#L152) but preseed_networking_setup.erb doesn't honor it (preseed_default_finish.erb does). Should it also be checking `static` parameter in several places in here too?

Reply:
Unless @mmoll knows more about why "static" option is added for preseed, I believe that "static" flag was initially added for gPXE/iPXE and it's not longer very tested and might not work at all. https://projects.theforeman.org/issues/470

Honoring it is probably good idea.